tests/kernel/interrupt/src/nested_irq.c

```diff
@@ -5,7 +5,8 @@ struct k_timer timer;
 
 #define ISR0_OFFSET 1
 #define ISR1_OFFSET 2
-
+/* ms is in milli seconds */
+#define DELAY_US(ms)  K_MSEC(ms) * USEC_PER_MSEC
```

Comment:
The DELAY_US(ms) might be misunderstood. I assume that you want to delay XX ms and macro returns value in us.
What about MS2US(ms) or just use the `K_MSEC(ms) * USEC_PER_MSEC` as an k_busy_wait() argument?

Reply:
Pedantry: that macro should be fully parenthesized so the multiplication binds as intended when used in an expression.  For example "delays_per_sec = 1000000 / DELAY_US()" will compute "(1000000/K_MSEC(ms)) * 1000" which is obvoiusly wrong.

Reply:
Agreed, incorporated both the reviews from @andyross and @pizi-nordic. Thanks.